server/src/main/java/com/paypal/selion/proxy/SeLionRemoteProxy.java

```diff
@@ -71,11 +73,12 @@
     private static final int MAX_SESSION_ALLOWED = 50;
     private static final int CONNECTION_TIMEOUT = 30000;
 
-    private volatile boolean forceShutDown;
+    private volatile boolean scheduledShutdown;
     private volatile int totalSessionsCompleted, totalSessionsStarted;
 
+    private boolean canForceShutdown, canAutoUpgrade, canViewLogs;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field canForceShutdown has the same name as a method](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1237916249/issues/source?bid=2928425&fileBranchId=2934495#l79)